src/java/org/apache/cassandra/db/guardrails/Guardrails.java

```diff
@@ -477,6 +478,21 @@ public final class Guardrails implements GuardrailsMBean
                      format("The modification to table %s has a timestamp %s before the minimum allowable %s threshold %s",
                             what, value, isWarning ? "warning" : "failure", threshold));
 
+    public static final MaxThreshold maximumReferencedIndexesWithNonParitionRestrictedQuery =
+    new MaxThreshold("maximum_referenced_indexes_for_non_paritition_key_restricted_query",
```

Comment:
nit: Make sure naming is consistent at the end w/ what's in the YAML and `Config`